server/controllers/index.js

```diff
@@ -1,11 +1,16 @@
 const express = require("express");
 const { getLocations, addLocation } = require("./locations");
+
+const getSuperUserRecords = require("./locations/getSuperUserRecords");
 const admin = require("./admin");
+
 const router = express.Router();
 
 router.get("/locations", getLocations);
 router.post("/locations", addLocation);
 
+router.get("/rep/:id/records", getSuperUserRecords);
```

Comment:
Remove the :id param please. I'm not keen on us passing privileged user IDs around openly. It should be possible to get the logged in user's details using the Express session once auth is implemented